doc/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx>=1.4
+sphinx_gallery
```

Comment:
got you @jasmainak. I first thought that this would be like a note for us, but yeah, we don't have to add this on.